test/run_test.py

```diff
@@ -1133,8 +1133,8 @@ def main():
         ]
         sys.path.remove('test')
 
-
-    selected_tests = reorder_tests(selected_tests)
+    if bool(os.environ.get('IN_CI')):
```

Comment:
Another options is
```suggestion
    if not options.bring_to_front:
```

Reply:
Could you explain the reasoning behind your use of `bool` here? i.e. what are the possible values of `IN_CI`, and which ones do you want to trigger this?

Reply:
It was done the same in common_utils.py I was following previous convension.

Reply:
it's probably low priority but this only checks for presence and not whether `IN_CI` is set to true or not, so it would still trigger on `IN_CI=0`, maybe more idiomatic would be to use `os.getenv("IN_CI", False)` without the cast

Reply:
yup. realized that... I felt the same but I guess I mainly trying to make sure it is consistent.

Reply:
also we can do: expose `IN_CI` similar to `IS_WINDOWS` from common_utils.py. (actually I like it this way)